package.json

```diff
@@ -17,9 +17,11 @@
     "prod": "npx tailwindcss -i ./src/styles/main.css -o ./public/output.css && webpack --config webpack.prod.js && node server/index.js"
   },
   "dependencies": {
+    "@hello-pangea/dnd": "^16.2.0",
```

Comment:
Alternative to react-beautiful-dnd but it is still not entirely clear at this point whether it fully solved the issue or having a useEffect to delay the react-beautiful-dnd will work.